README.md

```diff
@@ -143,34 +134,26 @@ After executing the above code, `response.StatusCode` should be `202` and you sh
 For more advanced cases, you can build the SendGridMessage object yourself with these minimum required settings:
 
 ```csharp
-using SendGrid;
-using SendGrid.Helpers.Mail;
 using System;
 using System.Threading.Tasks;
+using SendGrid;
+using SendGrid.Helpers.Mail;
 
-namespace Example
```

Comment:
In my opinion, these should be prefaced by accessors, especially if we're including `using` statements in the examples. Every class should be namespaced, even though it's kind of redundant in this example. `Main()` should be marked `private` and `class Program` should be marked `internal class Program`, for the sake of consistency.

Reply:
IMHO, the sample code should keep it as simple as possible.